.github/workflows/release.yaml

```diff
@@ -1,32 +1,16 @@
 # GitHub release workflow.
 name: Release
-run-name: Release ${{ github.ref_name }}${{ inputs.dry_run && ' (DRYRUN)' || '' }}
 on:
+  push:
+    tags:
+      - "v*"
   workflow_dispatch:
     inputs:
-      increment:
-        description: Preferred version increment (release script may promote e.g. patch to minor depending on changes).
-        type: choice
-        required: true
-        default: patch
-        options:
-          - patch
-          - minor
-          - major
-      draft:
-        description: Create a draft release (for manually editing release notes before publishing).
-        type: boolean
-        required: true
-        default: false
       dry_run:
-        description: Perform a dry-run release.
+        description: Perform a dry-run release (devel).
```

Comment:
You should mention that the ref must be annotated tagged if not doing this with dry run